lib/local-links-manager/import/local_authorities_importer.rb

```diff
@@ -38,6 +38,7 @@ def authorities_from_mapit
             Rails.logger.info("Updating authority #{mapit_la[:name]} (gss #{mapit_la[:gss]})")
             la.update_attributes!(
               name: mapit_la[:name],
+              snac: mapit_la[:snac],
```

Comment:
We can avoid `.tap` entirely because we don't actually use the `LocalAuthority` that we build/update for anything. Something like:

```ruby
la = LocalAuthority.where(gss: mapit_la[:gss]).first_or_initialize
verb = la.persisted? ? "Updating" : "Creating"
Rails.logger.info("#{verb} authority #{mapit_la[:name]} (gss #{mapit_la[:gss]})")

la.name = mapit_la[:name]
la.snac = mapit_la[:snac]
la.tier = mapit_la[:tier]
la.save!
```

There's something to be said for the block giving us some "encapsulation" around that work though, at the minor cost of understanding what `.tap` is doing (and why we're not using the block form of `first_or_initialize`).